types/google-maps-react/index.d.ts

```diff
@@ -94,15 +103,21 @@ export class HeatMap extends React.Component<any, any> {
 }
 
 export interface InfoWindowProps extends Partial<google.maps.InfoWindowOptions> {
-  google: typeof google;
-  map: google.maps.Map;
-  marker: google.maps.Marker;
+  google?: typeof google;
```

Comment:
I believe these properties are intended to be required.  Would you please look up the documentation and determine if they are optional?

Reply:
And should i fix it?

Reply:
> I believe these properties are intended to be required. Would you please look up the documentation and determine if they are optional?
And should i fix it?

Reply:
Right, you're supposed to pass them like this:

```tsx
public render() {
  return <div>
    <InfoWindow google={google} map={map} marker={marker} />
  </div>
}
```

And Typescript ought to tell you that you're missing one of them.

Reply:
Okey. Thanks. Fixed that